lab_5_scrapper/scrapper.py

```diff
@@ -168,6 +290,11 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        text = article_soup.find_all(style="text-align: justify;")
+        article = ''
```

Comment:
use more efficient concatenation strategy

Reply:
changed it